lms/djangoapps/course_api/blocks/tests/test_views.py

```diff
@@ -28,8 +29,23 @@ def setUpClass(cls):
         super(TestBlocksView, cls).setUpClass()
 
         # create a toy course
-        cls.course_key = ToyCourseFactory.create().id
+        cls.course = ToyCourseFactory.create()
+        cls.course_key = cls.course.id
         cls.course_usage_key = cls.store.make_course_usage_key(cls.course_key)
+        cls.chapter = ItemFactory.create(
+            category='chapter',
+            parent_location=cls.course.location,
+            due=datetime(3013, 9, 18, 11, 30, 00),
+            display_name="Chapter with due date",
+        )
+        cls.user = UserFactory.create()
+        cls.store.create_child(
```

Comment:
Can we create children using ItemFactory?
I tried creating child to `chapter` using ItemFactory (code below); but it was not appending it to children of `chapter`

```
ItemFactory.create(
    category='section',
    parent_location=cls.chapter.location,
    due=datetime(3013, 9, 18, 11, 30, 00),
    display_name="Section with due date",
)
```